app/javascript/dashboard/components/widgets/Thumbnail.vue

```diff
@@ -23,7 +23,7 @@
       src="~dashboard/assets/images/fb-badge.png"
     />
     <div
-      v-else-if="badge === 'Channel::WebWidget' && status === 'online'"
+      v-else-if="status === 'online'"
```

Comment:
Why was this removed?

Reply:
Seems like polluting the component, Added the same case through params to the Thumbnail component .

Reply:
The same is being used in a couple of places, repeating the logic doesn't seem good to me. I guess it is easier if the component can decide when to show status and when to show the badges